clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -278,7 +279,10 @@ def build_and_deploy_model(self,
             The number of replicas of the model to create. The number of replicas
             for a model can be changed at any time with
             :py:meth:`clipper.ClipperConnection.set_num_replicas`.
-
+        batch_size : int, optional
+            The user defined batch size.
+            The default value is -1, which means that the user did not define a specific batch size.
```

Comment:
Replace lines 284-285 with: "If the default value of -1 is used, Clipper will adaptively calculate the batch size for individual replicas of this model."